lib/IAMClient.js

```diff
@@ -52,6 +53,19 @@ class IAMClient {
         }
         this.accessKey = accessKey;
         this.secretKeyValue = secretKeyValue;
+        this.log = new Logger('IAMClient', {
+            level: 'info',
```

Comment:
Intended this method to handle that: https://github.com/scality/vaultclient/pull/136/files#diff-bc94f099c706d36757b47d7da8b97b92R67